templates/cli/lib/commands/init.js.twig

```diff
@@ -0,0 +1,261 @@
+const { messagingCreateTopic } = require("./messaging");
+const { functionsCreate } = require("./functions");
+const { databasesCreateCollection } = require("./databases");
+const ID = require("../id");
+const { localConfig, globalConfig } = require("../config");
+const {
+    questionsCreateFunction,
+    questionsCreateBucket,
+    questionsCreateMessagingTopic,
+    questionsCreateCollection,
+    questionsInitProject
+} = require("../questions");
+const { success, log, error, actionRunner, commandDescriptions } = require("../parser");
+const { accountGet } = require("./account");
+const { loginCommand } = require("./generic");
+
+const initProject = async () => {
+    let response = {};
+
+    try {
+        if (globalConfig.getEndpoint() === '' || globalConfig.getCookie() === '') {
+            throw '';
+        }
+        await accountGet({
+            parseOutput: false
+        });
+    } catch (e) {
+        log('You must login first')
+        await loginCommand()
+    }
+
+    const answers = await inquirer.prompt(questionsInitProject)
+    if (!answers.project || !answers.organization) {
+        process.exit(1)
+    }
+
+    if (answers.start === 'new') {
+        response = await projectsCreate({
+            projectId: answers.id,
+            name: answers.project,
+            teamId: answers.organization.id,
+            parseOutput: false
+        })
+
+        localConfig.setProject(response['$id'], response.name);
+    } else {
+        localConfig.setProject(answers.project.id, answers.project.name);
+    }
+
+    success();
+}
+
+const initBucket = async () => {
+    let response = {}
+    const answers = await inquirer.prompt(questionsCreateBucket)
+    if (!answers.bucket || !answers.id || !answers.fileSecurity) process.exit(1)
+
+    try {
+        response = await storageCreateBucket({
+            bucketId: answers.id,
+            name: answers.bucket,
+            fileSecurity: answers.fileSecurity.toLowerCase() === 'yes',
+            enabled: true,
+            parseOutput: false
+        })
+
+        localConfig.addBucket(response);
+        success();
+    } catch (e) {
+        error(e.getMessage ?? 'Unknown error occurred. Please try again');
+    }
+};
+
+const initCollection = async () => {
+    let response = {}
+    const answers = await inquirer.prompt(questionsCreateCollection)
+    if (!answers.database || !answers.collection || !answers.id || !answers.documentSecurity) process.exit(1)
+
+    try {
+        response = await databasesCreateCollection({
+            databaseId: answers.database,
+            collectionId: answers.id,
+            name: answers.collection,
+            documentSecurity: answers.documentSecurity.toLowerCase() === 'yes',
+            enabled: true,
+            parseOutput: false
+        })
+
+        localConfig.addCollection(response);
+        success();
+    } catch (e) {
+        error(e.getMessage ?? 'Unknown error occurred. Please try again');
```

Comment:
i dont think `getMessage` works, I think its `e.message`. Please test, and replace if needed. If replaced, please check all places with global search